### ISSUE TITLE
 False positive SC2119 an SC2120

### ISSUE THREAD

**Issue by bioinfornatics** [koalaman/shellcheck#1482]
Dear,

calling the special variable `$!` trigger false positive error.
This special variable is not bound to function statement. Indeed this variable return last pid.

Minimal case:

```bash
#!/usr/bin/env bash

get_pid(){
  ls ~/ &
  echo "$!"
}

bug_sc2119(){
  pid=$( get_pid )
}
```

Thanks for your works

Best regards

**Comment by austin987** [comment]
Did you try shellcheck.net? It works there (once you add `echo "$pid"` to the end to use the variable)

**Comment by bioinfornatics** [comment]
Thanks @austin987 

Indeed, it works fine on shellcheck.net 
So the version provided by CentOs 7, has this false positive warning which is fixed in a recent release.

Best regards
